app/routes/locations.py

```diff
@@ -0,0 +1,81 @@
+from flask import Blueprint, url_for, redirect, request, jsonify
+from app.models import db, Location, Amenities, Necessities
```

Comment:
will the import on line 2 work? i think it needs one more models like this:
from app.models.models import .....

Reply:
I am unsure. Arom is still working on the models at the time I made line 2, I expect line 2 may need to be updated if complications are encountered.